src/cache/ElasticacheRedis.spec.ts

```diff
@@ -1,142 +1,160 @@
-import { ElasticacheRedis, RedisClient } from './ElasticacheRedis';
-import { KeyValueCacheSetOptions } from './interface';
+import { afterEach, expect, jest, describe, it } from '@jest/globals';
```

Comment:
I ended up importing these because I wasn't getting accurate auto-completion types.  It also might have resolved the issue to add an explicit up to date `@types/jest` dev dependency, but I honestly also just prefer avoiding globals and explicitly declaring when possible.